src/main/java/study/StringAddCalculator.java

```diff
@@ -0,0 +1,46 @@
+package study;
+
+import java.util.Arrays;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+    public static String separator = ",|:";
```

Comment:
상수는 대문자 + snake case 문법을 따르는 것이 일반적인 java 컨벤션입니다 :)

```suggestion
    public static String SEPARATOR = ",|:";
```

Reply:
문자열을 상수로 분리해주신 것은 👍